packages/esm-patient-orders-app/src/components/order-details-table.component.tsx

```diff
@@ -374,27 +378,12 @@ const OrderDetailsTable: React.FC<OrderDetailsProps> = ({
           />
         </div>
         <span className={styles.rangeLabel}>{t('dateRange', 'Date range')}:</span>
-        <DatePicker
-          datePickerType="range"
-          dateFormat={'d/m/Y'}
-          value={''}
+        <OpenmrsDateRangePicker
+          value={dateRange}
           onChange={([startDate, endDate]) => {
             handleDateFilterChange([startDate, endDate]);
           }}
-        >
-          <DatePickerInput
-            id="startDatePickerInput"
-            data-testid="startDatePickerInput"
-            labelText=""
-            placeholder="dd/mm/yyyy"
-          />
-          <DatePickerInput
-            id="endDatePickerInput"
-            data-testid="endDatePickerInput"
-            labelText=""
-            placeholder="dd/mm/yyyy"
-          />
-        </DatePicker>
+        />
```

Comment:
Considering that we aren't providing an aria label, can you provide a `data-testid` for accessibility within tests?